cufflinks/plotlytools.py

```diff
@@ -1427,6 +1426,14 @@ def iplot(figure,validate=True,sharing=None,filename='',
 		auto_open=False
 
 	## Exports
+	if not offline.is_offline() or online:
+		try:
+			import chart_studio.plotly as py
+		except:
+			raise Exception("chart_studio is required: " \
+					"please run " \
```

Comment:
I would add a note here like “is required outside of offline mode” to encourage people to use offline mode :)

Reply:
Thank you! I updated it.